docs/development/documentation_guidelines.rst

```diff
@@ -17,25 +33,34 @@ To get started editing the docs:
 
    .. code:: sh
 
-      git clone https://github.com/freedomofpress/securedrop.git
+      git clone https://github.com/freedomofpress/securedrop-docs.git
+
 
 2. Install the dependencies:
 
+   .. include:: ../includes/virtualenv.txt  
+
    .. code:: sh
 
-      pip install --no-deps --require-hashes -r securedrop/requirements/python3/develop-requirements.txt
+      pip install --require-hashes -r requirements/requirements.txt
+
+
+3. Navigate to the ``docs/`` directory of the `SecureDrop docs repository <https://github.com/freedomofpress/securedrop-docs>`_ to make changes to the documentation rendered on https://docs.securedrop.org. 
 
-.. include:: ../includes/virtualenv.txt   
 
-3. Build the docs for viewing in your web browser:
+.. _build_the_docs:
+
+4. Build the docs for viewing in your web browser:
 
    .. code:: sh
 
       make docs
 
-You can then browse the documentation at http://127.0.0.1:8000/. As you make
-changes, the documentation pages will automatically rebuild in the browser
-window, so you don't need to refresh the page manually.
+   You can then preview the documentation at http://127.0.0.1:8000. The documentation pages will automatically rebuild in the browser
```

Comment:
I'm very open to revisiting this, but our current documentation guidelines (further in this chapter) suggest limiting the length of a line to about 80 characters unless long URLs or code blocks push it over the limit, so for now I would suggest applying the same line length throughout your changes.